osidb/models/affect.py

```diff
@@ -725,6 +751,14 @@ def bzsync(self, *args, **kwargs):
         self.affect.save(*args, **kwargs)
 
 
+# List of all states an Affect is considered open/not resolved
+AFFECTEDNESS_UNRESOLVED_RESOLUTIONS = {
+    Affect.AffectAffectedness.NEW: [
+        Affect.AffectResolution.NOVALUE,
+    ],
+    Affect.AffectAffectedness.NOVALUE: [],
```

Comment:
This affectedness option seems to me to never take an effect. Even though the affectedness would be NOVALUE, the resolutions - be it NOVALUE or any non-empty value - could not be in the empty list.

Reply:
makes sense, I'll remove this variable and move this check directly to the attribute then